packages/ember-glimmer/tests/integration/application/actions-test.js

```diff
@@ -48,3 +49,25 @@ moduleFor('Application test: actions', class extends ApplicationTest {
       });
   }
 });
+
+moduleFor('Rendering test: non-interactive actions', class extends RenderingTest {
+  getBootOptions() {
+    return { isInteractive: false };
+  }
+
+  [`@test doesn't attatch actions`](assert) {
+    this.registerComponent('foo-bar', {
+      ComponentClass: Component.extend({
+        actions: {
+          fire() {}
+        }
+      }),
+      template: `<button {{action 'fire'}}>Fire!</button>`
+    });
+
+    this.render('{{foo-bar}}');
+    let btnHTML = this.firstChild.innerHTML;
+
+    assert.equal(btnHTML, '<button>Fire!</button>');
```

Comment:
Might need to use one of the assert HTML helper methods to avoid issues across various browsers? (Just guessing though).